bitbucket-pipelines.yml

```diff
@@ -14,7 +14,8 @@ definitions:
         script:
```

Comment:
Wouldn't it be nicer to break down this script into several steps, like we did in other CIs?

Reply:
I would also fetch and install the dependencies, sonar-scanner and build-wrapper, first, before running the build, to mirror the scripts for other CIs.

Reply:
Alright, I think it is not worth the complexity.